libraries/ms-drivers/test/test_pca9539r_gpio_expander.c

```diff
@@ -308,3 +310,17 @@ void test_pca9539r_gpio_get_state_invalid_address(void) {
   Pca9539rGpioState state;
   TEST_ASSERT_EQUAL(STATUS_CODE_INVALID_ARGS, pca9539r_gpio_get_state(&address, &state));
 }
+void prv_test(const struct GpioAddress *address, void *context) {
+  GpioState state = GPIO_STATE_LOW;
+  gpio_get_state(address, &state);
+}
+void test_pca9539r_gpio_subscribe_interrupts(void) {
+  GpioAddress address = { .pin = VALID_PORT_0_PIN, .port = 0 };
```

Comment:
`VALID_PORT_0_PIN` is #define'd to refer to a pin on the PCA9539R gpio expander, rather than the normal GPIO of the STM32 that GpioAddresses refer to. I'd just use `0` as a generic "valid gpio pin". Also, you should be using `GPIO_PORT_A`, `GPIO_PORT_B`, etc (from gpio_mcu.h) as port values instead of literals like `0`.